frog/imports/api/activities.js

```diff
@@ -39,50 +50,50 @@ export const addActivity = (activityType, data = {}, id, groupingKey) => {
   }
 };
 
-export const duplicateActivity = activity =>
+export const duplicateActivity = (activity: ActivityDbT) =>
   Activities.insert({
     ...activity,
     _id: uuid(),
     data: { ...activity.data, name: activity.data.name + ' (copy)' }
   });
 
-export const addGraphActivity = params =>
+export const addGraphActivity = (params: Object) =>
   Activities.insert({
     ...params,
     graphId: params.graphId,
     createdAt: new Date(),
     _id: uuid()
```

Comment:
This is just a note for future study, but I read some article saying that storing uuids as strings in Mongo was extremely inefficient, and that it would be much more efficient to store it as a binary field or something - something to explore in the future.